### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by Blitheness** [Gaweph/randomsyntax-blog#2]
## Homepage
- [ ] Colour contrast on tags. Ratio is 2.7, should be at least 4.5 for WCAG 2 AA.
- [ ] Icons in footer do not have text for screen readers

# Blog posts
- [ ] Colour contrast in code blocks does not meet WCAG 2 AA standards. 
- [ ] Range input on Marching Squares post does not have a label for screen readers.
- [ ] Image in lightbox does not have alt text.

**Comment by Gaweph** [comment]
Thanks for this.  Blog has moved to a sub folder of the homepage repo.  I will fix these issues there.
